### PR TITLE
move jenkins file and docker file in build folder

### PR DESCRIPTION
**Pull request by sourabhsahu1**
## Description

Please describe the changes in your pull request in few words here.

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

Describe the steps required to test the changes proposed in the pull request.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: build/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+node() {
+    try {
```
Comment (tushar5526) [inline build/Jenkinsfile:2]: Hi, Please use the updated Jenkinsfile here as discussed in the call. Just copy and paste that Jenkinsfile from `samagra-devops` and change the `service name` to your `user-service`

Comment (tushar5526) [reply build/Jenkinsfile:2]: https://github.com/Samarth-HP/samarth-devops/blob/master/Jenkinsfile

**Review by tushar5526 — APPROVED** [review]

**Comment by tushar5526** [comment]
LGTM @ChakshuGautam

File: build/Jenkinsfile (hunk 1 above)
Comment (sourabhsahu1) [reply build/Jenkinsfile]: okay its done

**Pull request merged**